doc/htmldoc/models/models-main.rst

```diff
@@ -1,12 +1,152 @@
+.. _modelsmain:
+
 Models in NEST
 ==============
 
 
+What we mean by `models`
+------------------------
+
+Models in the context of NEST are C++ implmentations of mathematical equations that describe the characteristics and behavior of
+different types of neurons, syanpses, (and devices?), based on the relevant peer-reviewed publications for the model.
+
+We also use the term model in relation to network models (e.g., :ref:`microcircuit <toc_microcircuit>` and `multi-area model <https://inm-6.github.io/multi-area-model/>`_). These network models
+can be considerd a level of complexity higher than the neuron or synapse model. We do not focus on network models here.
+
+Find a model
+------------
+
 NEST provides a ton of models! Textbook standards like integrate-and-fire and Hodgkin-Huxley type models are available
 alongside high quality implementations of models published by the neuroscience community.
+The models are organized and autogenerated by keywords (e.g., adaptive threshold, conductance-based etc) in our directory. Models that contan a specific keyword will be listed under that word.
+
+* Discover :doc:`all the models in our directory <index>`.
+
+NESTML
```

Comment:
Maybe this paragraph as an admonition (``.. hint``)?

Reply:
done